wordpress-mobile/WordPress-iOS
# Updates WordPressUI.

Updated WordPressUI to try to resolve #14926.

This should not be merged until a new version of WPUI is released.

Since I wasn't able to reproduce the original issue, I'm not listing this as a fix in the readme file.

I do believe the changes in this PR should address the issues, though.

## Related PRs:

WPUI - https://github.com/wordpress-mobile/WordPressUI-iOS/pull/102

## To test:

1. Check that the "Me" icon looks fine.
2. Go to the notifications tab.
3. Scroll, change subsection, tap on notifications.

Make sure at all times the user icons look fine.

## Regression Notes

1. Potential unintended areas of impact

User images throughout the App.  Gravatar images.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

The are we can impact is what we're trying to fix.  Please check #14926.

I did a full logic review of our code and came up with an updated mechanism to ensure consistency in the images.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
